content/docs/introduction/first_steps.md

```diff
@@ -162,7 +162,7 @@ Now we restart our Prometheus server to activate our new job.
 
 Go to the expression browser and verify that Prometheus now has information
 about the time series that this endpoint exposes. Navigate to
-http://localhost:9090/graph and use the dropdown next to the "Execute" button to see a list of metrics this server is collecting. In the list you'll see a number of metrics prefixed with `node_`, that have been collected by the Node Exporter by our `node` job. For example, you can see the node's CPU usage via the `node_cpu` metric. 
+http://localhost:9090/graph and use the dropdown next to the "Execute" button to see a list of metrics this server is collecting. In the list you'll see a number of metrics prefixed with `node_`, that have been collected by the Node Exporter by our `node` job. For example, you can see the node's CPU usage via the `node_cpu_seconds_total` or `node_exporter_build_info` metric. 
```

Comment:
You can't see CPU usage with the node_exporter_build_info metric.

Reply:
Oh, you're right, I didn't look at the whole sentence wording. I'll send another fix.